lib/features/home/home_page.dart

```diff
@@ -20,6 +23,8 @@ class HomePage extends HookConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final txns = ref.watch(transactionsProvider);
+    final pfis = ref.watch(pfisProvider);
+
     // TODO(ethan-tbd): get balance from pfi, https://github.com/TBD54566975/didpay/issues/109
     final accountBalance = CurrencyUtil.formatFromDouble(0);
```

Comment:
not important for this PR but we should switch to using [this decimal lib](https://pub.dev/packages/decimal)